src/main/webapp/scripts/private-room-script.js

```diff
@@ -1,52 +1,198 @@
-		height: '390',
-		width: '640',
-		videoId: 'M7lc1UVf-VE',
-		events: {
-			'onReady': onPlayerReady,
-			'onStateChange': onPlayerStateChange
-		}
-	});
+    youtubePlayer = new YT.Player('player-div', 
+    {
+        playerVars: {
+            autoplay: 0,      // Don't autoplay the initial video
+            rel: 0,           //  Don’t show related videos
+            theme: "light",   // Use a light player instead of a dark one
+            controls: 1,      // Show player controls
+            showinfo: 0,      // Don’t show title or loader
+            modestbranding: 1 // No You Tube logo on control bar
+        },
+        events: {               
+            // Callback when onReady fires
+            onReady: onPlayerReady,
+            // Callback when onStateChange fires
+            onStateChange: onStateChange,
+        }
+    });
 }
 
 // The API will call this function when the video player is ready.
 function onPlayerReady(event) {
-	event.target.playVideo();
+    document.getElementById('player-div').style.borderColor = '#FF6D00';
+    loadRoomPlaylist();
 }
 
-// The API calls this function when the player's state changes.
-// The function indicates that when playing a video (state=1),
-// the player should play for six seconds and then stop.
-function onPlayerStateChange(event) {
-	if (event.data == YT.PlayerState.PLAYING && !isVideoFinished) {
-		setTimeout(stopVideo, 6000);
-		isVideoFinished = true;
-	}
+function stopVideo() {
+    youtubePlayer.stopVideo();
 }
 
-function stopVideo() {
-	youtubePlayer.stopVideo();
+// Log state changes
+function onStateChange(event) {
+    var state = "undefiend";
+    switch (event.data) {
+        case YT.PlayerState.UNSTARTED:
+            state= "unstarted";
+            break;
+        case YT.PlayerState.ENDED:
+            state = "ended";
+            break;
+        case YT.PlayerState.PLAYING:
+            state = "playing";
+            break;
+        case YT.PlayerState.PAUSED:
+            state = "paused";
+            break;
+        case YT.PlayerState.BUFFERING:
+            state = "buffering";
+            break;
+        case YT.PlayerState.CUED:
+            state = "video cued";
+            break;
+        default:
+            state = "unknown (" + event.data + ")";
+    }
+    console.log('onStateChange: ' + state);
+}
+
+async function displayChat() {
+    let response = await fetch('/chat');
+    let messages = await response.json();
+    const messageElement = document.getElementById('chat-messages');
+    messageElement.innerHTML = '';
+    for (message in messages) {
+      commentElement.appendChild(createParagraph(comments[comment]));
+    }  
 }
+
+function createParagraph(msgJson) {
+    var msg = JSON.parse(msgJson);
+    const paragraph = document.createElement('p');
+    paragraph.innerHTML = '<h4>';
+    paragraph.innerText = msg.sender;
+    paragraph.innerHTML = '</h4>';
+    paragraph.innerText = msg.message;
```

Comment:
paragraph.innerHTML for all statements. Also, can this just be one line using ``

Reply:
This was not supposed to be included. I have now removed it from the file